src/awesomeSezzle.js

```diff
@@ -675,25 +675,30 @@ class AwesomeSezzle {
 		return currency || 36;
 	}
 
-	calculateInterest(price, term, APR) {
+	calculateMonthlyWithInterest(price, term, APR) {
 		if(APR > 0){
 			const rate = (APR/100)/12;
 			const numerator = price * rate * Math.pow(1+rate, term);
 			const denominator = Math.pow(1+rate, term)-1;
 			const interestPayment = numerator/denominator
 			return interestPayment;
```

Comment:
if we remove `toFixed(2)` from the function call, shouldn't we add `toFixed(2)` for the case when APR > 0?

Reply:
It is happening in `addDelimiters` function. Technically don't even need .toFixed(2) on line 686.

Reply:
Then, we probably should delete it on the line 686, because in this case, the format functions get a string from the
`calculateMonthlyWithInterest` when APR is 0 and a float when APR > 0